workshop/nanos/nano3.md

```diff
@@ -0,0 +1,112 @@
+# 🛠 Nano Activity 3: Setup VS Code the Right Way for Vue (beginner)
+
+| **Project Goal**            |  The goal of this nano is to learn how to properly setup VS Code for Vue development                                                                                                                                   |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **What you’ll learn**       | You'll learn how to setup VS Code for development with Vue.                                                                                            |
+| **Tools you’ll need**       | A modern browser like Chrome or Firefox, Access to [Visual Studio Code Editor](https://code.visualstudio.com/download)
+| **Time needed to complete** | 15 minutes
+
+
+# We're going to learn how to setup VScode the right way for Vue
+
+In this activity we are going to walkthrough how to configure your VS Code workspace for appropiate development with Vue. As a developer working with Vue, it is a great deal to msetup your work tool to increase productivity whereby you use tool that already do most of the work you neccessarily don't need to bother about.
+
+## Get Started
+
+In order to start developing Vue applications. A required step is to first setup your workspace or code editor for development and this nano is going to highlight how to setup VS Code for Vue.
+
```

Comment:
You might want to add a bit here outlining the elements that are necessary specifically for a Vue developer. Why, Vetur, for example? What are the pain points specific to Vue?